articles/quickstart/native/flutter/files/main.md

```diff
@@ -0,0 +1,19 @@
+---
+name: main.dart
+language: dart
+---
+
+```dart
+// Instantiate the SDK once in your app
+final auth0 = Auth('${account.namespace}', '${account.clientId}');
+
+// Redirect to Auth0 Universal Login for authentication
+final result = await auth0.webAuthentication.login();
```

Comment:
Isn't `webAuthentication` a method now?

Reply:
Yes. Much of this was written before the latest changes were applied.